README.md

```diff
@@ -1,31 +1,56 @@
 # Ghostery Browser Extension
 
-This repository is home to all Ghostery Browser Extensions for various platforms.
+> A powerful privacy extension that blocks ads, prevents trackers, and accelerates website loading
 
-## Folder structure
+Ghostery is a powerful Tracker & Adblocker extension for Chrome with over 100 million downloads. Block ads, stop trackers, and speed up websites. Privacy at its best.
 
-* Extension `v10.x` for **Chrome**, **Edge**, **Opera** and **Safari**: [extension-manifest-v3](/extension-manifest-v3)
+* Block all ads on websites, including YouTube and Facebook, to focus on the information that matters.
+* Stop trackers from collecting your personal data.
+* Automatically remove intrusive cookie pop-ups and express dissent to online tracking.
+* Get detailed tracker information on any website you visit, including the number of trackers, their type, and the company operating them.
+* Preview tracker information on search engine result pages to make informed choices.
+* Inspect the largest database of trackers, updated fast and reliably for all users.
 
 ## Links
 
-+ [Website](https://www.ghostery.com/)
-+ [Broken Page Reports](https://github.com/ghostery/broken-page-reports/)
-+ [Support](https://www.ghostery.com/support)
-+ [X, formerly Twitter (@ghostery)](https://twitter.com/ghostery)
-+ [Privacy Policy](https://www.ghostery.com/about-ghostery/browser-extension-privacy-policy/)
+* [Website](https://www.ghostery.com/)
+* [Broken Page Reports](https://github.com/ghostery/broken-page-reports/)
+* [Support](https://www.ghostery.com/support)
+* [X, formerly Twitter (@ghostery)](https://twitter.com/ghostery)
+* [Privacy Policy](https://www.ghostery.com/about-ghostery/browser-extension-privacy-policy/)
 
 ## Additional Open Source Ghostery Projects
 
-+ [Ghostery Desktop Browser](https://github.com/ghostery/user-agent-desktop)
-+ [Ghostery iOS Browser](https://github.com/ghostery/user-agent-ios)
-+ [Ghostery Android Browser](https://github.com/ghostery/user-agent-android)
+* [Ghostery Desktop Browser](https://github.com/ghostery/user-agent-desktop)
+* [Ghostery iOS Browser](https://github.com/ghostery/user-agent-ios)
+* [Ghostery Android Browser](https://github.com/ghostery/user-agent-android)
+
+## Development
+
+To start the local version of the extension in Chrome, run the following commands:
+
+```bash
+npm i && npm start
```

Comment:
please make those two separate steps and suggest usage of `npm ci` instead of `npm i`